hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/utils/AccessorBasedUsages.java

```diff
@@ -180,7 +182,11 @@ private void estimateCryptoUpdate(
             SigUsage sigUsage, TxnAccessor accessor, BaseTransactionMeta baseMeta, UsageAccumulator into) {
         final var cryptoUpdateMeta = accessor.getSpanMapAccessor().getCryptoUpdateMeta(accessor);
         final var cryptoContext = opUsageCtxHelper.ctxForCryptoUpdate(accessor.getTxn());
-        cryptoOpsUsage.cryptoUpdateUsage(sigUsage, baseMeta, cryptoUpdateMeta, cryptoContext, into);
+        // We use the SDK default auto-renew period for auto-association slot lifetime if account auto-renew is disabled
+        final var explicitAutoRenewSlotLifetime =
```

Comment:
Why is this here instead of, I dunno, `GlobalDynamicProperties`?

Just seems like a unfortunate place to hide this explicit number.

Reply:
Moved this to `GlobalDynamicProperties`.